### DISCUSSION TITLE
Disable cronjobs for redudent setup

### DISCUSSION THREAD

**Discussion by TheNerdyHamster** [semaphoreui/semaphore#840] [Q&A]
Hi,

I am currently working on setting up a secondary instance of semaphore (a copy of main instance, that updates once a day) which is just a backup in case of the main instance goes down.

This is resulting in that all cronjobs from the main instance is also running on the secondary instace, which might result in changed playbooks being runned etc.

Is there any recommended solution for disabling cronjobs currently in semaphore?

**Comment by fiftin** [comment]
Why not to stop semaphore on the second instance? I don't know how to do this another way.

**Reply by TheNerdyHamster** [reply]
Stopping the second instance was my first thought, but in this case there are other "non" tech people that are running playbooks from time to time, and incase the primary instace goes down, then someone dosen't need to login and turn on the backup.
Two solutions/workaround I have thought of so far:
1. Remove cron information after insert (which would kinda work iguess).
2. Adding some sort of swithced based of configuration value that either runs the cronjobs or not. (Havn't fully checked out how much work is required yet), but would prob be something I could contribute with.

Hope's this makes some sense!
